build/post-app.js

```diff
@@ -9,22 +10,25 @@ exports.execute = function(options, webapp) {
   // Updates manifest.webapp
   require('./post-manifest').execute(options, webapp);
 
-  require('./multilocale').execute(options, webapp);
+  require('./multilocale').execute(options, webapp).then(function() {
```

Comment:
I haven't tried it myself yet, but maybe we could do here what @ochameau  suggested in https://bugzilla.mozilla.org/show_bug.cgi?id=1116613#c7 ?  See also:

https://github.com/mozilla-b2g/gaia/blob/1ff734dba9812f34782b8dabb1ce3b2bdb7ca0e3/build/app.js#L130-L133
https://github.com/mozilla-b2g/gaia/blob/1ff734dba9812f34782b8dabb1ce3b2bdb7ca0e3/build/svoperapps.js#L315-L321